src/cascadia/TerminalControl/TerminalControl.vcxproj

```diff
@@ -10,6 +10,8 @@
     <SubSystem>Console</SubSystem>
     <!-- sets a bunch of Windows Universal properties -->
     <OpenConsoleUniversalApp>true</OpenConsoleUniversalApp>
+    <!-- C++/WinRT sets the depth to 1 if there is a XAML file. We need it to be 3. -->

```

Comment:
We need it to be 3 because why? Leave reason.

Reply:
@DHowett helped me solved this issue, so I think he could provide an accurate explanation for this :)

Reply:
@DHowett-MSFT too so he can see it on the work one as well.

Reply:
```suggestion
    <!-- C++/WinRT sets the depth to 1 if there is a XAML file in the project.
         Unfortunately for us, we need it to be 3. When the namespace merging
         depth is 1, Microsoft.Terminal.TerminalControl becomes "Microsoft",
         and our WinMD file becomes "Microsoft". Because WinRT is very
         namespace-driven, this winmd is consider as speaking for the entire
         Microsoft namespace. This is, obviously, not great. None of our other
         projects compile properly when they depend on this "Microsoft.winmd."
    -->
```

Reply:
Added